File: code.java
import java.util.*; 
import java.io.*;

class Main {  
  public static String TimeConvert(int num) { 
  
    int fp = num /60;
    int sp = num % 60;
    
       
    return Integer.toString(fp) + ":" + Integer.toString(sp);
    
  } 
  
  public static void main (String[] args) {  
    // keep this function call here     
    Scanner s = new Scanner(System.in);
    System.out.print(TimeConvert(s.nextLine())); 
  }   
  
}

File: code.cpp
#include <iostream>
#include <string>
#include <sstream>
using namespace std;

string TimeConvert(int num) { 

  // code goes here
  stringstream ss;
  ss << num/60 <<":"<<num%60;
  return ss.str();
            
}

int main() { 
  
  // keep this function call here
  cout << TimeConvert(gets(stdin));
  return 0;
    
}
